src/main/java/se/liu/ida/hefquin/cli/modules/ModFederation.java

```diff
@@ -287,4 +290,14 @@ protected URI verifyExpectedURI( final String uriString ) {
 		return uri;
 	}
 
+	protected boolean verifyValidVocabMappingFile(final String pathString ) {
+		File f = new File(pathString);
+		if ( f.exists() ){
+			// TODO: .nt file?
+			return f.isFile();
```

Comment:
If it is not an actual file, you return false here. In this case, with the current implementation that calls this function, there will be no error whatsoever. However, there should be one (similar to the case in which the file/directory does not exist at all (i.e., the else branch below).